client/my-sites/backup/wpcom-upsell.tsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * External dependencies
+ */
+import React, { ReactElement } from 'react';
+import { translate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import DocumentHead from 'components/data/document-head';
+import Main from 'components/main';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import FormattedHeader from 'components/formatted-header';
+import PromoSection, { Props as PromoSectionProps } from 'components/promo-section';
+import PromoCard from 'components/promo-section/promo-card';
+import PromoCardCTA from 'components/promo-section/promo-card/cta';
+import useTrackCallback from 'lib/jetpack/use-track-callback';
+import Gridicon from 'components/gridicon';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+import MaterialIcon from 'components/material-icon';
+import WhatIsJetpack from 'components/jetpack/what-is-jetpack';
+
+/**
+ * Asset dependencies
+ */
+import JetpackBackupSVG from 'assets/images/illustrations/jetpack-backup.svg';
+import './style.scss';
+
+const trackEventName = 'calypso_jetpack_backup_business_upsell';
+
+const promos: PromoSectionProps = {
+	promos: [
+		{
+			title: translate( 'Jetpack Scan' ),
+			body: translate(
+				'Automated scanning and one-click fixes to keep your site ahead of security threats.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **7** times:
`translate( 'Scan gives you automated scanning and one-click fixes to keep your site ahead of security threats.' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-section-backup-upsell)

Reply:
This sounds like a good translation to re-use.

Reply:
Yep, we can reuse that. I'll be honest and say I trimmed it in the original designs because it gave me a better/tighter fit. :D